social/backends/open_id.py

```diff
@@ -350,6 +413,20 @@ def validate_and_return_id_token(self, id_token):
             self.remove_nonce(nonce_obj.id)
         else:
             raise AuthTokenError(self, 'Incorrect id_token: nonce')
+
+    def validate_and_return_id_token(self, jws):
+        """
+        Validates the id_token according to the steps at
+        http://openid.net/specs/openid-connect-core-1_0.html#IDTokenValidation.
+        """
+        try:
+            # Decode the JWT and raise an error if the sig is invalid
+            id_token = JWS().verify_compact(jws.encode('utf-8'), self.get_jwks_keys())
```

Comment:
These long lines should likely be at or below 80 characters to match the rest of the file/app